.eslintrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
Out of curiosity, does running Prettier in the repo catch this file?

Reply:
currently no, if we made this change to the `package.json` script then we'd include it:

```diff
diff --git a/package.json b/package.json
index d89d75738..047720d17 100644
--- a/package.json
+++ b/package.json
@@ -41,7 +41,7 @@
     "rollup.watch": "rollup --watch --config",
     "license": "node scripts --license",
     "prettier": "npm run prettier.base -- --write",
-    "prettier.base": "prettier \"./({bin,scripts,src,test}/**/*.{ts,tsx,js,jsx})|bin/stencil\"",
+    "prettier.base": "prettier \"./({bin,scripts,src,test}/**/*.{ts,tsx,js,jsx})|*.js|bin/stencil\"",
     "prettier.dry-run": "npm run prettier.base -- --list-different",
     "test": "jest --coverage",
     "test.analysis": "cd test && npm run analysis.build-and-analyze",
```

there might be a more elegant way to do that!

doing so would also format our `jest.config.js` and `rollup.config.js` files, resulting in pretty minimal changes:

```diff
diff --git a/jest.config.js b/jest.config.js
index 2071bc01a..095f25159 100644
--- a/jest.config.js
+++ b/jest.config.js
@@ -57,5 +57,5 @@ module.exports = {
   testRegex: '/(src|scripts)/.*\\.spec\\.(ts|tsx|js)$',
   // TODO(STENCIL-307): Move away from Jasmine runner for internal Stencil tests, which involves re-working environment
   // setup
-  testRunner: 'jest-jasmine2'
+  testRunner: 'jest-jasmine2',
 };
diff --git a/rollup.config.js b/rollup.config.js
index 9a3d58d01..db6708815 100644
--- a/rollup.config.js
+++ b/rollup.config.js
@@ -1,7 +1,7 @@
 const { createBuild } = require('./scripts/build/build.js');
 const { getOptions } = require('./scripts/build/utils/options.js');
 
-module.exports = function(args) {
+module.exports = function (args) {
   const opts = getOptions(__dirname, {
     isProd: !!args['config-prod'],
     isCI: !!args['config-ci'],
```

I think we might want to do that, just for consistency

Reply:
Yeah I like that - let's go that route!